cmd/mattermost/commands/channel.go

```diff
@@ -117,6 +117,7 @@ func init() {
 	ChannelCreateCmd.Flags().Bool("private", false, "Create a private channel.")
 
 	MoveChannelsCmd.Flags().String("username", "", "Required. Username who is moving the channel.")
+	MoveChannelsCmd.Flags().Bool("remove-deactivated-users", false, "Automatically remove any deactivated users from the channel before moving it.")
```

Comment:
0/5, but should this just be a discrete command? The move operation could fail with a clear message and suggest this as a pre-flight step.

Reply:
That's an interesting idea, but I think it's not worth it because I'm not sure of any reason that people are needing this functionality outside of this command currently. Not opposed to adding it in future as a standalone command if it becomes apparent there *is* a need for it though.